cfgov/unprocessed/css/organisms/item-introduction.less

```diff
@@ -35,11 +35,28 @@
 */
 
 .o-item-introduction {
+    // visually approximates 60px
```

Comment:
I'm not sure we are still doing eyeball adjustments; maybe @jimmynotjim / @Scotchester  can enlighten me.